docs/getting-started/local.rst

```diff
@@ -153,4 +153,30 @@ This plugin is invoked under the hood via. the ``--python_gapic_out`` switch.
   where it expects to find protos, and *order matters*. In this case,
   the common protos must come first, and then the path to the API being built.
 
+
+Generating Samples
```

Comment:
Why is this info duplicated? We should have a SPOT and then as many references to it as we need. I know RST supports inclusion, but GitHub's implementation does not. Is that the issue? I've run into that and I have some general thoughts about a possible workaround.

Reply:
RST supports inclusion. The issue I was trying to avoid was 

<duplicated samplegen text>
<docker or local specific text>
<duplicated samplegen text>

Includes are a somewhat coarse-grained mechanism for solving that problem, and I don't think RST has macros. Solved by conjoining the duplicated text into a contiguous block.